pkg/reducer/examples/counter/main.go

```diff
@@ -20,5 +20,7 @@ func reduceCounter(_ context.Context, keys []string, reduceCh <-chan reducer.Dat
 }
 
 func main() {
-	reducer.NewServer(reducer.ReducerFunc(reduceCounter)).Start(context.Background())
+	reducer.NewServer(reducer.ReducerCreatorFn(func() reducer.Reducer {
```

Comment:
This is too complex to use, can we add a help function like below?
 
```golang
func SimpleReducer(func (_ context.Context, keys []string, reduceCh <-chan reducer.Datum, md reducer.Metadata) reducer.Messages) ReduceCreator {
 ///
}
```

Reply:
```golang
type dummyReducerCreator struct {
	f func(ctx context.Context, keys []string, reduceCh <-chan Datum, md Metadata) Messages
}

func (d dummyReducerCreator) Create() Reducer {
	return ReducerFunc(d.f)
}

// NewDummyReducerCreator creates a dummy ReducerCreator.
func NewDummyReducerCreator(f func(ctx context.Context, keys []string, reduceCh <-chan Datum, md Metadata) Messages) ReducerCreator {
	return dummyReducerCreator{f: f}
}
```

Reply:
`ReducerFunc` can be private in this case.

Reply:
Start function will be:

```golang
reducer.NewServer(SimpleReducerCreator(reduceCounter)).Start(context.Background())
```

Reply:
I like this change :)